src/components/Swap/Toolbar/ToolbarTradeSummary.tsx

```diff
@@ -0,0 +1,119 @@
+import { t } from '@lingui/macro'
+import { formatCurrencyAmount, formatPriceImpact, NumberType } from '@uniswap/conedison/format'
+import { CurrencyAmount, Token } from '@uniswap/sdk-core'
+import Column from 'components/Column'
+import Row from 'components/Row'
+import Tooltip from 'components/Tooltip'
+import { PriceImpact } from 'hooks/usePriceImpact'
+import { Slippage } from 'hooks/useSlippage'
+import { AlertTriangle, Icon, Info } from 'icons'
+import { ReactNode } from 'react'
+import { InterfaceTrade } from 'state/routing/types'
+import styled from 'styled-components/macro'
+import { Color, ThemedText } from 'theme'
+
+import { SwapInputOutputEstimate } from '../Summary'
+import { PriceImpactWarningTooltipContent } from './Caption'
+
+const TradeSummaryColumn = styled(Column)`
+  border-bottom: 1px solid ${({ theme }) => theme.outline};
+  border-top: 1px solid ${({ theme }) => theme.outline};
+  margin: 0 1em;
+  padding: 0.75em 0;
+`
+
+const TradeAttributeName = styled(ThemedText.Body2)`
+  color: ${({ theme, color }) => color ?? theme.secondary};
+`
+
+const TradeAttributeValue = styled(ThemedText.Body2)`
+  color: ${({ theme, color }) => color ?? theme.primary};
+`
+
+interface SummaryRowProps {
+  name: string
+  value: string
+  color?: Color
+  tooltip?: {
```

Comment:
Do you think separating this out into `nameTooltip` and `valueTooltip` would be more readable? I'm unsure.